src/components/List/List.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
-import bem from '../../utils/bem';
-import styles from './List.scss';
+import { bem } from '../../utils/bem';
 import { LIST_NAVIGATION_DIRECTIONS, ENTER_KEY } from '../../constants';
-import scrollIntoViewIfNeeded from '../../utils/scrollIntoViewIfNeeded';
+import { scrollIntoViewIfNeeded } from '../../utils/scrollIntoViewIfNeeded';
```

Comment:
maybe we should create a `index.ts` in utils and export this one from there as well